hack/tools/go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/sigstore/sigstore/hack/tools
+
+go 1.17
```

Comment:
Isn't everything else go 1.16?

Reply:
i will move to 1.16, but i guess that does not hurt stay as is

Reply:
done